shared/logging.properties

```diff
@@ -0,0 +1,8 @@
+handlers = com.microsoft.azure.appservice.logging.AsyncFileHandler, java.util.logging.ConsoleHandler
+
+# AzMon logging
+com.microsoft.azure.appservice.logging.AsyncFileHandler.level = FINE
+com.microsoft.azure.appservice.logging.AsyncFileHandler.directory = /var/log/diagnosticLogs
```

Comment:
>/var/log/diagnosticLogs [](start = 68, length = 23)

can you check the permissions on the log files getting created ?

Reply:
read access to owner, group, others